lwgrp.spec

```diff
@@ -0,0 +1,92 @@
+Name:		lwgrp
+Version:	1.0.2
+Release:	3.1%{?dist}
+Summary:	Light-weight Group Library for MPI process groups 
+
+License:	BSD
+URL:		https://github.com/LLNL/lwgrp
+Source0:	https://github.com/LLNL/lwgrp/releases/download/v%version/lwgrp-%version.tar.gz
+Patch1:		lwgrp-sover.patch
+BuildRequires: automake
+
+%description
+The light-weight group library defines data structures and collective
+operations to group MPI processes as an ordered set.  Such groups are
+useful as substitutes for MPI communicators when the overhead of
+communicator creation is too costly.  For example, certain sorting
+algorithms recursively divide processes into subgroups as the sort
+algorithm progresses.  These groups may be different with each
+invocation, so that it is inefficient to create and destroy
+communicators during the sort routine.
+
+%global openmpi_ver openmpi3
+
+%if 0%{?fedora}
+%global openmpi_ver openmpi
+%endif
+
+
+%package %{openmpi_ver}
+Summary: Light-weight Group Library for MPI process groups
+BuildRequires: %{openmpi_ver}-devel
+
+
+%description %{openmpi_ver}
+The light-weight group library defines data structures and collective
+operations to group MPI processes as an ordered set.
+
+
+%package %{openmpi_ver}-devel
+Summary: Light-weight Group Library for MPI process groups -- development files
+Requires: %{name}-%{openmpi_ver}%{_isa} = %version-%release
+Requires: %{openmpi_ver}-devel
+
+
+%description %{openmpi_ver}-devel
+Development files for %{name}-%{openmpi_ver}.
+
+%prep
+%setup -q
+%patch1 -p1
+autoreconf
+
+%build
+%if "%{openmpi_ver}" == "openmpi"
+%{_openmpi_load}
+%else
+%{_openmpi3_load}
+%endif
+%configure --disable-static --includedir=$MPI_INCLUDE --libdir=$MPI_LIB
+make %{?_smp_mflags}
+
+
+%install
+%make_install
+
+
+%files %{openmpi_ver}
+%license LICENSE.TXT
+%doc README
+%_libdir/%{openmpi_ver}/lib/liblwgrp.so.*
+%exclude %_libdir/%{openmpi_ver}/lib/liblwgrp.la
```

Comment:
Why not be consistent with what you did here https://github.com/daos-stack/libcircle/blob/9fd3cacdece4708476c8fe2e3a0a613ba367465a/libcircle.spec#L109